src/services/api-service.js

```diff
@@ -0,0 +1,49 @@
+import axios from 'axios';
+import { history } from '../history'
+
+export class ApiService {
+    constructor(options = {}) {
+        this.client = axios.create({
+          baseURL: "http://localhost:8000/api",
+          ...options
+        });   
+        this.client.interceptors.response.use(
+          this.handleSuccessResponse,
+          this.handleErrorResponse
+        );
+        this.unauthorizedCallback = () => {}; 
+    }
+
+    addHeader(headers) {
+      this.client.defaults.headers = headers;
+    }
+
+    handleSuccessResponse(response) {
+      history.push('/user-dashboard');
```

Comment:
Sto bi na svakom success responseu hendlala redirect? :D

Reply:
Greska, ispravila sam to sad kod register-a. :D